### PR TITLE
Main elements and styling

### PR DESCRIPTION
**Pull request by flpfar**
This branch contains the following parts that compose the project:

- Every HTML content like texts, links, icons, etc.
- Header fixed for small devices
- Main contents inside bootstrap grids
- Single elements as cards
- Footer using bootstrap flex
- Media queries for different screen sizes
- Appropriate readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
-# newsweek-clone
\ No newline at end of file
+# Clone - "Newsweek" Website
+
+> This is a clone of ["Newsweek" Website (TNW)](https://www.newsweek.com/), created to practice responsive layouts with bootstrap and media queries using a mobile first approach.
+
```
Comment (majovanilla) [inline README.md]: Good description!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1232 @@
...
+    <!-- FONTS -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lora:400,700%7CRoboto+Condensed:400,700&display=swap">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <title>Newsweek - Clone</title>
+  </head>
+  <body>
+      <!-- HEADER -->
+      <header class="d-flex align-items-center">
+        <div class="header-top d-flex align-items-center align-items-lg-end justify-content-lg-between w-100">
```
Comment (majovanilla) [inline index.html:21]: Excellent use of bootstrap classes.

**Review by majovanilla — APPROVED** [review]
Hey @flpfar , 👋

## You did an *amazing*🌟 ✨ job on this project! 👊🏻  💯
It looks professional and meets all the requirements.

-  Your PR message and Readme Description are very good.

- You made great use of bootstrap classes.

- The responsiveness of your project is behaving as expected.

- You added the hover effect to links, which makes your page looks great (not everyone does this and I always suggest it, cause it makes your project look more professional)

- You are using semantic HTML.

You did everything as expected and that's why I will approve it right away.

### **STATUS: PROJECT APPROVED!**

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 

Happy coding 🤓

--Maria Reyes

**Comment by flpfar** [comment]
Thank you for reviewing my code, @majovanilla ! I'm going to merge this branch then :+1:

**Pull request merged**